acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/configuration/BesuNodeFactory.java

```diff
@@ -149,6 +160,16 @@ public BesuNode createArchiveNodeWithAuthenticationOverWebSocket(final String na
             .build());
   }
 
+  public BesuNode createArchiveNodeWithAuthenticationUsingJwtPublicKeyOverWebSocket(
```

Comment:
NIT: createNodeWithAuthenticationUsingJwtPublicKeyOverWebSocket

Also, from an AT point of view, we could probably have one method `createNodeWithAuthenticationUsingJwtPublicKey()` that would include the json rpc and the websocket config. I don't think there is value in having them separate.